lib/travis/build/data.rb

```diff
@@ -203,6 +203,11 @@ def repository
         data[:repository] || {}
       end
 
+      def allowed_repositories
+        return data[:allowed_repositories] if data[:allowed_repositories]
```

Comment:
This can be more succinctly (and still readably)

    data[:allowed_repositories] || [github_id]

Reply:
Nice, thank you!